Docs/Configuration.tex

```diff
@@ -928,34 +937,30 @@ \subsection{Delete Properties}\label{acpipropsdelete}
 \item
   \texttt{Enabled}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
-  \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: This ACPI table will not be removed unless set to
-  \texttt{true}.
+  \textbf{Failsafe}: \texttt{false} (Do not remove this ACPI table)\\
```

Comment:
Should be consistent with delete/remove maybe?

Reply:
Removed additional clarification of failsafe line since true/false is binary and once what `true` does is explained, can forego explaining `false`.

Applied to other similar instances as well.